code/modules/events/holiday/halloween.dm

```diff
@@ -40,6 +40,10 @@
 	icon_state = "coffincookie"
 	crafting_complexity = FOOD_COMPLEXITY_2
 
+/obj/item/food/cookie/sugar/spookycoffin/Initialize(mapload, seasonal_changes = FALSE)
+	// Changes default parameter of seasonal_changes to FALSE, pass to parent
+	return ..()
```

Comment:
imo you should either make it explicit (like `return ..(mapload, \*seasonal_changes =*\ TRUE)`) or just make it a var on sugar cookies